c2corg_ui/templates/base.html

```diff
@@ -141,6 +143,7 @@
          module.constant('imageUrl', '${image_url}');
          module.constant('discourseUrl', '${discourse_url}');
          module.constant('authUrl', '${request.route_path('auth')}');
+         module.constant('textFormatingUrl', '${request.route_path('articles_view_id', id=151910)}');
```

Comment:
I do not want to add it in each edit template, here is the place that we will seach for it. Note that ``route_path`` is really cheap.